main.go

```diff
@@ -320,6 +352,7 @@ func generateExportOptionsPlist(exportMethodStr, teamID string, uploadBitcode, c
 	} else {
 		options := exportoptions.NewNonAppStoreOptions(exportMethod)
 		options.CompileBitcode = compileBitcode
+		options.DistributionBundleIdentifier = productBundleID
```

Comment:
This option was introduced with Xcode 12 afaik, so it only needs to be added in that case.

Reply:
That's right 👍

Reply:
So an Xcode version check is neccessary in that case?

Reply:
I do not think that this would cause the build to fail, this is just an extra entry in a plist but the user will not be able to export a clip if he chooses a pre-Xcode12 stack. Should we display a warning or maybe make the build fail if the user selects `clip` with Xcode <12?

Reply:
It is not only used for app clip export only, as far as I can see, so in that case needs tested with older Xcodes also that the additional parameter is not breaking anything. In the xcode-archive step we check if Xcode 12 is used and only then add this extra DistributionBundleIdentifier parameter, so it makes sense to do the same here for consistency.

` Should we display a warning or maybe make the build fail if the user selects clip with Xcode <12?`  It sounds good.

Reply:
Done